ancar/portfolio/src/main/webapp/style.css

```diff
@@ -16,7 +16,7 @@ h1 {
     background-color: lightcoral;
     color: white;
     border-style: none;
-    width: 1800px;
+    width: 1700px;
```

Comment:
hardcoding pixel values instead of specifying ratios can be harmful long term.. I know it is not new in this change, just be aware for now.

Reply:
thank you, great advice for future! :)